# A representation of pose composed of position and orientation

geometry_msgs/Point position
Euler orientation